src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -546,7 +546,7 @@ final Charset charset() {
     protected String outputTimestamp;
 
     /**
-     * The algorithm to use for selecting which files to compile. The default values are implemented in `incrementalCompilationConfiguration()`
+     * The algorithm to use for selecting which files to compile,The default values are implemented in {@link #incrementalCompilationConfiguration()}
```

Comment:
The problem is that, in my understanding, this Javadoc will appear as the [plugin documentation](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#useIncrementalCompilation). Readers will have no idea what this `incrementalCompilationConfiguration()` is. Even if we were generating classic Javadoc, this link would be broken because this field is protected and the `incrementalCompilationConfiguration()` method is package-private.

We have to remove this link from this javadoc. It was a tip for maintainers, not for users. If we want to keep this comment, it should be a comment on a line starting with `//` below the Javadoc.